### ISSUE TITLE
Credentials file contents improperly modified

### ISSUE THREAD

**Issue by Nuru** [Noovolari/leapp#260]
The `$AWS_SHARED_CREDENTIALS_FILE` (by default `~/.aws/credentials`) is a standard, _shared_ file which is [intended to hold sensitive configuration information](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where) like access keys. It is fine that Leapp stores session keys in this file, as that is its intended use. (The fact that Leapp does not honor the `$AWS_SHARED_CREDENTIALS_FILE` setting is a separate issue.)

The problem is that, unlike the `$AWS_CONFIG_FILE` (default `~/.aws/config`), the `$AWS_SHARED_CREDENTIALS_FILE` is a shared file meant to be dynamically modified by applications that are maintaining short-lived credentials. It is not a file that Leapp "owns". Therefore, Leapp should _never_ remove entries it did not add. Leap should only update the file using a read-modify-write strategy where it reads the file, modifies the entries it is controlling (adding or removing sessions), and leaving the remaining entries unmodified. 

As a side note, I disagree with #98: Leapp should NOT import existing information from the `$AWS_SHARED_CREDENTIALS_FILE` into Leapp on first run or really ever. To the extent Leapp wants to import existing configuration at startup, it should limit itself to config files. 

**Leapp Version**
0.11.0

**To Reproduce**

1. Install Leap, configure it, use it, quit it
2. Put some content in the `$AWS_SHARED_CREDENTIALS_FILE`
3. Run Leapp
4. Quit Leapp

**Expected behavior**
The `$AWS_SHARED_CREDENTIALS_FILE` should contain the same information as it had after step 2.

**Actual behavior**
The `$AWS_SHARED_CREDENTIALS_FILE` is empty.

**Desktop (please complete the following information):**
 - OS: macOS
 - OS Version 11.6.5
 - Leapp Version 0.11.0

**Comment by RamazanKara** [comment]
I agree. This makes migrating to Leapp from other solutions like awsume, which are relying on the credential file, really a pain. Also leapp does not provide a non interactive solution so it is not really feasible for a full overtake of the credential file.
